docs/debug_tests.rst

```diff
@@ -0,0 +1,81 @@
+.. _topics-debug_tests:
+
+===========
+Debug Tests
+===========
+
+The test results go in ``results/<date>—<test_number>``. For results from a particular test, look for ``results/<date>—<test_number>/test_class_name/<test_method_name>/`` directory. The ``test_log.debug`` file will contain the log output from the test, and logs of services used in the test will be in ``service_name/node_name`` sub-directory.
+
+If there is not enough information in the logs, you can re-run the test with ``—no-teardown`` argument::
+
+    ducktape dir/tests/my_test.py::TestA.test_a —no-teardown
```

Comment:
If you use `.. sourcecode:: <language>` on these blocks, you can get syntax highlighting to make them more readable.